Misc/NEWS.d/next/Library/2022-05-28-19-41-02.gh-issue-93096.qjUyRG.rst

```diff
@@ -0,0 +1,3 @@
+Command-line :mod:`mimetypes` started to return 1 instead of 0 for
+abruptions and 2 instead of 1 for incorrect command line parameters.
```

Comment:
abruptions? Perhaps:
```suggestion
Command-line :mod:`mimetypes` started to exit with 1 instead of 0
on failure and 2 instead of 1 for incorrect command line parameters.
```